src/helpers/metrics.ts

```diff
@@ -38,3 +40,49 @@ new client.Gauge({
     this.set((await db.queryAsync(`SELECT SUM(total) as count FROM reqs`))[0].count as any);
   }
 });
+
+// Metrics PushGateway test
+// @todo To be moved to snapshot-metrics package after live testing
+const INSTANCE = process.env.METRICS_INSTANCE;
+const JOB_NAME = process.env.METRICS_JOB_NAME ?? 'prometheus';
+const PUSHGATEWAY_URL = process.env.METRICS_PUSHGATEWAY_URL;
+
+async function sleep(time: number) {
+  return new Promise(resolve => {
+    setTimeout(resolve, time);
+  });
+}
```

Comment:
I feel better to move it to utils like other repos

Reply:
This will be moved to @snapshot-sentry, like the entire new block of code